packages/non-core/bundle-visualizer/server.js

```diff
@@ -33,21 +33,38 @@ function getStatBundles() {
     f.absolutePath.endsWith(".stats.json");
 
   // Read the stat file, but if it's in any way unusable just return null.
-  const readOrNull = file => {
+  function readOrNull(file) {
     try {
       return JSON.parse(fsReadFileSync(file, "utf8"));
     } catch (err) {
       return null;
     }
-  };
+  }
+
+  const {
+    staticFiles,
+    staticFilesByArch,
+  } = WebAppInternals;
+
+  const files = [];
+
+  if (staticFilesByArch) {
```

Comment:
Total drive-by comment, but you might be able to `DRY` this up a bit with something like:

```js
if (staticFilesByArch || staticFiles) {
  const sourceFiles = staticFilesByArch || { '': staticFiles };
  Object.keys(sourceFiles).forEach(arch => {
    const staticFiles = sourceFiles[arch];
    Object.keys(staticFiles).forEach(path => files.push(staticFiles[path]));
  });
}
```